src/routes/Apps/App/TrainDialogs.tsx

```diff
@@ -125,9 +125,9 @@ function getColumns(intl: InjectedIntl): IRenderableColumn[] {
                         <span data-testid="train-dialogs-description">
                             {trainDialog.description
                                 || <>
-                                    <span data-testid="train-dialogs-first-input">{firstInput ? firstInput : ''}</span>
+                                    <span /*data-testid="train-dialogs-first-input"*/>{firstInput ? firstInput : ''}</span>
```

Comment:
Good to see this. Minor, Would prefer to remove the comment also.

Reply:
Okay, I removed the comments.